src/Ombi/ClientApp/src/app/my-nav/my-nav.component.scss

```
@@ -1,260 +1,326 @@
-        align-items:center;
-        font-weight: 700;
-        font-size:36px;
-        padding:40px 20px;
-        height:auto;
-        max-width: 300px;
-        display: flex;
-        white-space: normal;
+        text-decoration: none;
+        display: block;
+        line-height: 1.2;
         word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)